docs/api_requests/block-io-engine.md

```diff
@@ -49,10 +51,13 @@ If a block device is configured with the `Async` io_engine on a host kernel
 older than 5.10.51, the API call will return a 400 Bad Request, with a
 suggestive error message.
 
-## Recommendations
+## Performance considerations
 
-In order to get the higher disk IO throughput, it is recommended to use the
-`Async` io_engine on hosts that support it.
+An important thing to note is that the performance is strictly tied to the host
```

Comment:
I would just say The performance is strictly.. Drop the first part which doesn't add up information.

Reply:
done